### ISSUE TITLE
Podman -v or --volume flag isn't valid

### ISSUE THREAD

**Issue by nicolasduminil** [containers/podman#19846]
### Issue Description

The podman -v or --volume flags raise "Unknown flag" error

### Steps to reproduce the issue

Steps to reproduce the issue

    $ podman pod create -p 80:8080 --name mypod --volume ./html:/var/www/html:z

### Describe the results you received

    Error: unknown flag: --volume

### Describe the results you expected

A new pod be created

### podman info output

```yaml
$ podman version
Version:      3.4.4
API Version:  3.4.4
Go Version:   go1.18.1
Built:        Thu Jan  1 01:00:00 1970
OS/Arch:      linux/amd64
```

### Podman in a container

No

### Privileged Or Rootless

Rootless

### Upstream Latest Release

No

### Additional environment details

Additional environment details

### Additional information

Additional information like issue happens only occasionally or issue happens with a particular architecture or on a particular setting

**Comment by giuseppe** [comment]
`--volume` is not a valid flag for a pod.

You can specify it to `podman create` or `podman run`

**Comment by giuseppe** [comment]
I've closed the issue but please feel free to comment further

**Comment by nicolasduminil** [comment]
@giuseppe What do you mean by ==volume is not valid for a pod ? Do you mean that the following command isn't expected to work ?

    $ podman pod create -p 8080:8080 --name mypod --volume ./html:/var/www/html:z

Have a look at _Podman in Action_, Chapter 4.2, page 80. Now, if you want to close issues, be my guest but the most important is to  fix them.

**Comment by vrothberg** [comment]
Yes, `podman create --volume` exists but Podman v3.4.4 is too old.  The flag was added by https://github.com/containers/podman/pull/11409 which made it into Podman 4.0.0.

**Comment by vrothberg** [comment]
@nicolasduminil, apologies, I wrote ` create --volume` but meant `pod create --volume`.  The latter was added by https://github.com/containers/podman/pull/11409 and is only available with Podman v4.0 and after.

> The documentation instructs to install podman by running `apt-get install podman` and, doing that, podman 3.4.4 gets installed.

That seems like an issue in the Podman book.  @rhatdan can you take a look?  I guess the book requires Podman 4.0.

More recent version of Ubuntu ship newer versions of Podman.  Maybe that's an alternative.

**Comment by rhatdan** [comment]
Yes the book assumes podman 4.0 or later.
